composer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "woocommerce/woocommerce",
+	"description": "An eCommerce toolkit that helps you sell anything. Beautifully.",
+	"homepage": "https://woocommerce.com/",
+	"type": "wordpress-plugin",
+	"license": "GPL-3.0-or-later",
+	"prefer-stable": true,
+	"minimum-stability": "dev",
+	"require": {
+		"php": ">=7.2",
+		"woocommerce/woocommerce-sniffs": "^0.1.2"
+	},
+	"require-dev": {
+	    "squizlabs/php_codesniffer": "^3.5"
+	},
+	"scripts": {
+		"phpcs-pre-commit": [
+			"phpcs -s -p -n"
```

Comment:
This command takes forever for me, and shows an immense amount of errors, do we want to pass in only the changed files?
I often tag on this `$(git diff --name-only --diff-filter=AM trunk | grep .php$)` locally, so that `phpcs` only checks the changed files.

Reply:
AFAIK, Husky will pass the changed files as command line parameters to this command. I'll check this again.